test/custom-elements/samples/global-styles/_config.js

```diff
@@ -0,0 +1,3 @@
+export default {
+	dev: true
```

Comment:
Should probably also test for `dev: false` as well, right?

Reply:
I clone it from another test. I suppose this is for warnings, isn't it? Could this file be removed completely?

Reply:
I simplified the test, now it just checks only related to global styles functionality